frontend/packages/console-shared/src/selectors/node.ts

```diff
@@ -18,6 +18,9 @@ export const getNodeRoles = (node: NodeKind): string[] => {
   );
 };
 
+export const isLabelPresent = (node: NodeKind) =>
```

Comment:
@parthdhanjal 
```
export const isLabelPresent = (node: NodeKind, label: string) : boolean =>
  _.has(node, ['metadata', 'labels', label]);
```

Reply:
I would also argue that this function probably belongs in `frontend/packages/console-shared/src/selectors/common.ts`. Labels are common to all k8s resources, so this could be reused elsewhere. Also, I think 'hasLabel' is probably a more concise and accurate name for this util function.

```ts
export const hasLabel = <A extends K8sResourceKind = K8sResourceKind>(resource: A, label: string) : boolean =>
  _.has(resource, ['metadata', 'labels', label]);
```